changelogs/fragments/329-postgresql_add_connect_params_field.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- postgresql_* - Add ``connect_params`` parameter dict to allow any additional libpg connection parameters.
```

```suggestion
- postgresql_* - add the ``connect_params`` parameter dict to allow any additional ``libpg`` connection parameters (https://github.com/ansible-collections/community.postgresql/pull/329).
```
To make it consistent with other changelog entries